### DISCUSSION TITLE
Discussion in name of Yatopia

### DISCUSSION THREAD

**Discussion by TheArcaneBrony** [kennytv/Yaptapia#2] [General]
Hello, I wanted to contact you around your criticism towards Yatopia. I do appreciate your time and effort to point out the flaws in how we work, and we are trying to improve this bit by bit. If you'd like to help us out in fixing our workflow, this would be greatly appreciated. If there is anything we can do, let us know!

**Comment by mdcfe** [comment]
🌈

**Reply by TheArcaneBrony** [reply]
This applies to you too, if there's anything we can do to fix compatibility with EssentialsX feel free to tell us what to change.

**Reply by mdcfe** [reply]
We did and it was blindly rebutted without proof (parts of that conversation are linked in this repository). I have better uses of my time than repeating the exact same conversation I already had with folks in your Discord.

**Reply by TheArcaneBrony** [reply]
Well yes, I know what the main issues are (lack of testing, lack of checks, ...), I meant more like, specifics, like x line of code/patch is causing issues.

EDIT: I just realised said specifics would be way too much effort, though any info would help.

**Comment by JRoy** [comment]
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟥🟥
🟥🟥🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟥🟥
🟥🟥🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟥🟥
🟥🟥🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟦🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥 ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎‎ ‎🟥🟥🟥🟥🟥🟥
🟥🟥🟥🟥🟥🟥 ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ 🟥🟥🟥🟥🟥🟥
(amoungus)

**Reply by Chew** [reply]
this reply is awfully sus

**Comment by kennytv** [comment]
As you said yourself, reviewing your ported patches in detail would be too much effort, plus, the implications are already given by commit history and such. As noted in the page, it's not just about improving, but also being transparent about instability and ongoing issues as they appear and that they *are* something you have to expect on a semi-regular basis; prime examples for that are the sidelining of the "minor" issues when people make less thoughtful comments, *only* referencing the year-old corruption issue. While that argument is a very weak one on its own, you basically ignore all the little/medium issues coming up (see the original page) and only address the one, flawed argument (tho that's admittedly partly caused by those commenters). The second one would be the block-state issue, as outlined on the main page.

**Reply by TheArcaneBrony** [reply]
On the transparency note, I have added a little statement in the readme ([see PR](https://github.com/YatopiaMC/Yatopia/pull/510)).
